go/storage/mkvs/checkpoint/restorer.go

```diff
@@ -101,6 +103,7 @@ func (rs *restorer) RestoreChunk(ctx context.Context, idx uint64, r io.Reader) (
 		rs.pendingChunks = nil
 		rs.currentCheckpoint = nil
 
+		_ = rs.ndb.FinishMultipartInsert(false)
```

Comment:
Shouldn't the argument be `true` as this is an abort? Could also just call `rs.AbortRestore(ctx)` here (and remove the separate lock/nilification)?